modules/migration-excluding-resources.adoc

```diff
@@ -32,25 +32,15 @@ metadata:
 spec:
   disable_image_migration: true <1>
   disable_pv_migration: true <2>
+  additional_excluded_resources: <3>
+  - foo
```

Comment:
Don't use "fubar" or variations of it.

<img width="892" alt="image" src="https://user-images.githubusercontent.com/83772007/158804812-71fdaf14-732d-423b-a0d5-b6f9d2116882.png">

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f